packages/contentful--create-contentful-app/src/utils.ts

```diff
@@ -77,11 +77,13 @@ export function normalizeOptions(options: CLIOptions): CLIOptions {
   }
 
   if (currentMutuallyExclusiveOptions.length > 1) {
-    warn(
-      `Options ${highlight('--source')}, ${highlight('--example')}, ${highlight(
-        '--typescript'
-      )} and ${highlight('--javascript')} are mutually exclusive, using ${choice(fallbackOption)}.`
-    );
+    const paramsString = `Options ${currentMutuallyExclusiveOptions
```

Comment:
nit: You can use Intl to make this more readable

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/format

```
new Intl.ListFormat('en', { style: 'long', type: 'conjunction' }).format(currentMutuallyExclusiveOptions);
```
^^ misses `--` prefix and highlighting

Reply:
Oh, nice, I completely forgot about this stuff.